{
  "comments": [
    {
      "key": {
        "uuid": "4670fe4a_1cc7e45a",
        "filename": "tests/test_zuul_scheduler.py",
        "patchSetId": 4
      },
      "lineNbr": 768,
      "author": {
        "id": 24
      },
      "writtenOn": "2019-05-13T10:22:37Z",
      "side": 1,
      "message": "Which covers Zuul bug fix in 2.5.1wmf9 :]",
      "range": {
        "startLine": 768,
        "startChar": 40,
        "endLine": 768,
        "endChar": 64
      },
      "revId": "0f0aecfb695e668786ac9847dfed63f79d8fcca6",
      "serverId": "e9e9afe9-4712-486d-8885-f54b72dd1951",
      "unresolved": false
    }
  ]
}